python_modules/dagster/dagster/_utils/caching_instance_queryer.py

```diff
@@ -383,3 +396,19 @@ def get_current_minutes_late_for_key(
             used_data_times=used_data_times,
             available_data_times={key: evaluation_time for key in used_data_times},
         )
+
+
+def _extract_tags(record: Optional[EventLogRecord]) -> Mapping[str, str]:
```

Comment:
I think we should make this a `materialization_tags` property on `EventLogRecord`. Or add a `materialization` property to `EventLogRecord` and pull the tags off of it in the caller.